src/snowflake/connector/cursor.py

```diff
@@ -132,7 +132,7 @@ def from_column(cls, col: dict[str, Any]):
             col["name"],
             FIELD_NAME_TO_ID[
                 col["extTypeName"].upper()
-                if "extTypeName" in col
+                if col.get("extTypeName")
```

Comment:
@sfc-gh-kwagner please take a look since you changed to use `extTypeName` in https://github.com/snowflakedb/snowflake-connector-python/pull/966. Is there any concern for this change?

Reply:
This change is fine

Reply:
Thank you!